web/src/InstallationFinished.jsx

```diff
@@ -39,21 +39,27 @@ import {
   EOS_CHECK_CIRCLE as SectionIcon
 } from "eos-icons-react";
 
-const Actions = ({ onRestart }) => (
-  <Button isLarge variant="secondary" onClick={onRestart}>
-    Restart Installation
-  </Button>
+const Actions = ({ onRestart, onReboot }) => (
+  <>
+    <Button isLarge variant="secondary" onClick={onRestart}>
+      Restart Installation
+    </Button>
+    <Button isLarge variant="primary" onClick={onReboot}>
+      Reboot
```

Comment:
NP: I'd use "Reboot now" or "Reboot system" to be more explicit, but taking into account that the screen already has an explanatory text it is ok as it is.